src/darwin/Framework/CHIP/CHIPDeviceController.mm

```diff
@@ -65,9 +63,11 @@ @interface CHIPDeviceController ()
  * The delegate queue where delegate callbacks will run
  */
 @property (readonly, nonatomic) dispatch_queue_t delegateQueue;
-@property (readonly) chip::DeviceController::ChipDeviceController * cppController;
+@property (readonly) chip::Controller::DeviceCommissioner * cppController;
```

Comment:
rename to `cppCommissioner`?

Reply:
Maybe, the whole class ('CHIPDeviceController.mm`) can be refactored/renamed. We can provide a controller and a commissioner API, similar to CPP layer.

Reply:
For time being we are not seeing a usecase for separating Controller from Commissioner in iOS framework. Let's leave the filenames as is. I am not neutral to changing the variable name to `cppCommissioner`. Let me know if you strongly feel about it.